specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/NetworkWatcherFlowLogList.json

```diff
@@ -18,6 +18,7 @@
               "targetResourceId": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Network/networkSecurityGroups/vm5-nsg",
               "targetResourceGuid": "00000000-0000-0000-0000-000000000000",
               "storageId": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Storage/storageAccounts/wzstorage002",
+              "enabledFilteringCriteria": "srcIP=158.255.7.8 || dstPort=56891",
```

Comment:
>enabledFilteringCriteria [](http://example.com/codeflow?start=15&length=24)

Same comment from a different PR on the same issue.
[https://github.com/Azure/azure-rest-api-specs-pr/pull/18515](https://github.com/Azure/azure-rest-api-specs-pr/pull/18515)

This seems to be very tricky to get the correct input for the user.

enabledFilteringCriteria is said to be for filtering flowlogs. Can it go on the query parameters as $filter ?

Reply:
The NRP includes checks to verify the format of the passed value in this field. We are expecting an expression representing some conditions using relational and logical operators. Case sensitivity and extraneous spaces are disregarded in these specifications. Additional details and formatting specifications will be outlined in the documentation.